### PR TITLE
Nuevas Skins para Cyborgs

### PR DESCRIPTION
**Pull request by NervalARS**

### CODE REVIEW

File: code/__HELPERS/global_lists.dm [hunk 1]
```diff
@@ -47,8 +47,8 @@ var/list/mannequins_
 var/global/list/all_species[0]
 var/global/list/all_languages[0]
 var/global/list/language_keys[0]					// Table of say codes for all languages
-var/global/list/whitelisted_species = list("Human") // Species that require a whitelist check.
-var/global/list/playable_species = list("Human")    // A list of ALL playable species, whitelisted, latejoin or otherwise.
+var/global/list/whitelisted_species = list("Human", "Kidan") // Species that require a whitelist check.
```
Comment (fernandos13) [inline code/__HELPERS/global_lists.dm:50]: this got a lot of "kidan" stuff. not to mention a lot of other code that got nothing to do with sprites.

File: code/game/jobs/job/medical.dm [hunk 2]
```diff
@@ -76,7 +76,7 @@ Your second loyalty is to your career with Moebius corp, and to your coworkers i
 
 	access = list(
 		access_moebius, access_medical_equip, access_morgue, access_surgery, access_chemistry, access_virology,
-		access_genetics
+		access_genetics, access_maint_tunnels
```
Comment (fernandos13) [inline code/game/jobs/job/medical.dm:79]: although i would love medical and moebius to have maint. this was discussed in the past. and so far. agreed to not be a thing

File: code/game/objects/items/weapons/manuals.dm [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*********************MANUALS (BOOKS)***********************/
-
+/* Commented so that it loads Hispania manuals.dm
```
Comment (fernandos13) [inline code/game/objects/items/weapons/manuals.dm:2]: this is literally from another codebase

File: code/hispania/game/objects/items/weapons/kitchen.dm [hunk 4]
```diff
@@ -0,0 +1,34 @@
+/obj/item/weapon/cheese_grater
```
Comment (fernandos13) [inline code/hispania/game/objects/items/weapons/kitchen.dm:1]: i love this recipe.. but its also not in the pr document

File: code/hispania/modules/mob/language/station.dm [hunk 5]
```diff
@@ -0,0 +1,19 @@
+/datum/language/kidan
+	name = LANGUAGE_KIDAN
+	desc = "The noise made by rubbing its antennae together is actually a complex form of communication for Kidan."
```
Comment (fernandos13) [inline code/hispania/modules/mob/language/station.dm:3]: more race thing even a new language. i think you merged either in the wrong codebase. or the wrong branch

**Comment by TheShown911** [comment]
Si?

File: code/hispania/modules/reagents/reagent_containers/food/cans.dm [hunk 6]
```diff
@@ -0,0 +1,7 @@
+/obj/item/weapon/reagent_containers/food/drinks/cans/nuevabebida
+    name = "Coltrina"
+    desc = "Es una bebida con un gusto citrico pero dulce"
```
Comment (fernandos13) [inline code/hispania/modules/reagents/reagent_containers/food/cans.dm:3]: in spanish needs to be translated. "a drink with a citric taste, but pretty sweet"

File: code/modules/organs/internal/liver.dm [hunk 7]
```diff
@@ -45,7 +46,7 @@
 
 
 	if(owner.chem_effects[CE_ALCOHOL_TOXIC])
-		take_internal_damage(owner.chem_effects[CE_ALCOHOL_TOXIC], prob(90)) // Chance to warn them
+		take_internal_damage(owner.chem_effects[CE_ALCOHOL_TOXIC] * alcohol_mod_damage, prob(90)) // Chance to warn them
```
Comment (fernandos13) [inline code/modules/organs/internal/liver.dm:49]: whats this for ?? that new race? or for everyone?

**Review by fernandos13 — CHANGES_REQUESTED** [review]
i think you got the wrong branch. or wrong fork bud. 
or you are missing a LOT of documentation

**Comment by Firefox13** [comment]
@fernandos13 holy shit Fernandos, stop reviewing it, it's definitely not intended here, you're a madman

**Comment by TheShown911** [comment]
Basado

File: code/hispania/game/objects/items/weapons/kitchen.dm (hunk 4 above)
Comment (TotoMC-13) [reply code/hispania/game/objects/items/weapons/kitchen.dm:1]: want me to do it but in eris folder?

**Pull request closed without merging**